integration-tests/test-apps/standalone/dotenv-vercel/.env.vault

```diff
@@ -0,0 +1,25 @@
+#/-------------------.env.vault---------------------/
```

Comment:
This is the start of the test app that I build to use the `standalone-analyzer` on. This one happens to use an example from `dotenv` `vercel`.